src/debug/telemetry/Kconfig

```diff
@@ -9,3 +9,8 @@ config SOF_TELEMETRY
 	  systick_info measurement which measures scheduler task performance and may
 	  slightly affect overall performance.
 
+config SOF_TELEMETRY2_SLOT
+	bool "Enable SOF telemetry2 slot in debug window"
```

Comment:
what on earth is telemetry**2**?

Reply:
I think @lgirdwood 's original plan was to move the current telemetry node under a chunk in this new system, e.g. not to waste full 4k just for the current telemetry structure. That of course would need agreement from Windows side and some coordination, so for the time being I just decided to call it telemetry2.

Reply:
The precisions are even less clear than the original, sorry. you're drowning me in details I didn't even know I didn't need :-)